backend/benefit/applications/enums.py

```diff
@@ -37,3 +37,18 @@ def resolve_organization_type(cls, company_form):
             return OrganizationType.COMPANY
         else:
             return OrganizationType.ASSOCIATION
+
+
+class AttachmentType(models.TextChoices):
+    EMPLOYMENT_CONTRACT = "employment_contract", _("employment contract")
+    PAY_SUBSIDY_DECISION = "pay_subsidy_decision", _("pay subsidy decision")
+    COMMISSION_CONTRACT = "commission_contract", _("commission contract")
+    EDUCATION_CONTRACT = "education_contract", _(
+        "education contract of the apprenticeship office"
+    )
+    HELSINKI_BENEFIT_VOUCHER = "helsinki_benefit_voucher", _("helsinki benefit voucher")
+
+
+class AttachmentRequirement(models.TextChoices):
```

Comment:
Should this be used somewhere or will it be used later?

Reply:
Yes, this actually belongs to another PR that is about attachment business rules, removed for now.